controllers/app.controllers.js

```diff
@@ -1,7 +1,14 @@
-const { getEndpointsFromFile } = require("../models/app.models");
+const { getEndpointsFromFile, getTopicsFromDatabase } = require("../models/app.models");
 
 exports.getApi = (req, res, next) => {
   getEndpointsFromFile().then((endpoints) => {
     res.status(200).send({ endpoints });
   });
 };
+
+exports.getTopics = (req, res, next) => {
+  getTopicsFromDatabase().then(topics => {
+    console.log(topics)
+    res.status(200).send({topics})
+  })
+}
```

Comment:
In your controller you are using the next parameter but then not using it anywhere as an argument, it's a good idea to write some code to handle your next declaration.